apps/test/unit/code-studio/pd/workshop_dashboard/workshopTest.jsx

```diff
@@ -22,7 +22,7 @@ describe('Workshop', () => {
   it('renders', () => {
     const wrapper = shallow(
       <Workshop
-        params={{workshopId: fakeWorkshop.id}}
+        params={{workshopId: fakeWorkshop.id.toString()}}
```

Comment:
For my understanding, why did you move this?

Reply:
I feel like I'm going to learn a lot about JS testing from you :)

Reply:
The react props enforce that this field is actually a number on the "workshop" objects, it only needs to be a string for the workshopId fields, so when you pass the whole workshop, it should be a number by default it think

Reply:
I learned a lot about testing when updating the tests 😅 I haven't used enzyme and chai before

Reply:
haha you could have fooled me!

Reply:
Ooh I see what you're saying about workshop ID, interesting -- no idea why we use integers in one place and strings in another, but that's for another day. Nice cleanup!